deltachat-ios/MessageKit/Layout/FileMessageSizeCalculator.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import UIKit
+
+open class FileMessageSizeCalculator: MessageSizeCalculator {
+
+    var defaultFileMessageCellWidth: CGFloat {
+        switch UIApplication.shared.statusBarOrientation {
+        case .landscapeLeft, .landscapeRight:
+            return UIScreen.main.bounds.size.width * 0.66
```

Comment:
I haven't tested it yet, but this is probably not gonna work with Ipads on split-screen-mode.

We should maybe address this in another PR, since this will probably be something that needs to be fixed on MessageSizeCalculator).